.github/workflows/generate.yml

```diff
@@ -0,0 +1,35 @@
+name: generate
+
+on:
+  push:
+    branches: [ master, main ]
+  pull_request:
+    paths-ignore:
+      - .gitlab-ci.yml
+      - .github/workflows/test.yml
+      - .github/workflows/docker.yml
+      - .github/workflows/docker-pres_xml.yml
+
+jobs:
+  test-linux:
+    name: Test on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.METANORMA_CI_PAT_TOKEN || github.token }}
+          submodules: true
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/xml2rfc
+          key: xml2rfc
+          restore-key: xml2rfc
+
+      - uses: actions-mn/setup@master
+
+      - run: metanorma site generate . -o published -c metanorma.yml --agree-to-terms
```

Comment:
Same here.

Reply:
Yep, unfortunatelly it's broken because of https://github.com/metanorma/metanorma-cli/pull/224